demo/upload_file_limit_test/run.py

```diff
@@ -0,0 +1,24 @@
+import gradio as gr
+
+with gr.Blocks() as demo:
+    gr.Markdown("""
+    # ⬆️📁 max_file_size test
+    Components have a max file size of 1kb (disabling upload). The error modal should pop up when a file is uploaded.          
```

Comment:
```suggestion
    This Gradio app limits uploads to be 1b or lower. The error modal should pop up when a file is uploaded larger than that limit          
```

Reply:
Is it 1b or 1kb? The playwright config suggests 1b

Reply:
Fixed!